src/authn/authn.ts

```diff
@@ -262,16 +224,12 @@ function logInLoadPreferences (context) {
          */
         function differentOrigin () {
           return (
-            window.location &&
-            window.location.origin + '/' !== preferencesFile.site().uri
+            window.location && `${window.location.origin}/` !== preferencesFile.site().uri
```

Comment:
I agree it's not easy to read, but changing the logic to using `!` would mean to return something that is always true-ish, which is not a good fit with how the function is used.

Reply:
Hmm, I don't think I understand how the logic would be changed? Would this not work?

```suggestion
            window.location &&
            window.location.origin! + '/' !== preferencesFile.site().uri
```

Reply:
Sorry, I misread the logic. Actually, on closer look, I think it makes sense to simply say ``${window.location.origin}/` !== preferencesFile.site().uri` since `window.location.origin` should always be set. (TypeScript and the default types for `window` confirms this.)

Reply:
:+1: